packages/grid/src/features/standard-table/components/StandardTableRowList.tsx

```diff
@@ -91,8 +98,8 @@ export const StandardTableRowList = React.memo(function StandardTableRowList<
         <StandardTableRow
           alwaysVisible={disableInfiniteList || sortedItems.length < 30}
           item={item}
-          itemIdList={itemIdList}
-          key={itemIdList[index]}
+          idListForEnabledItems={idListForEnabledItems}
+          key={idListForEnabledItems[index]}
```

Comment:
`index` from `sortedItems` shouldn't be used in `idListForEnabledItems` since they are different lists and could differ

Reply:
Oups, nice catch!